pkg/controller/operatingsystemconfig/os-common/README.md

```diff
@@ -0,0 +1,82 @@
+# [Gardener Extension for OS Configurations](https://gardener.cloud)
```

Comment:
File should be removed (the content is misleading, no actual controller is implement here, only library functions)

Reply:
The entire file? it provide information about how to use the library.

Reply:
OK, agreed, didn't see that. 👍